packages/engine/bin/cli/src/experiment.rs

```diff
@@ -148,7 +148,7 @@ async fn run_experiment_with_manifest(
             proto::EngineStatus::Logs(sim_id, logs) => {
                 for log in logs {
                     if !log.is_empty() {
-                        info!("[{sim_id}]: {log}");
+                        info!(target: "behaviors", "[{experiment_id}][{sim_id}]: {log}");
```

Comment:
Are we sure the "Logs" variant of EngineStatus is always from user-code. If so I think we should rename the variant to `BehaviorLogs` or something similar. Also documentation on the enum.